extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/build/TracesBuildConfig.java

```diff
@@ -24,8 +24,8 @@ public class TracesBuildConfig {
      * and either this value is true, or this value is unset.
      */
     @Deprecated
-    @ConfigItem(defaultValue = "${quarkus.opentelemetry.tracer.enabled:true}")
```

Comment:
Please update the documentation because `${quarkus.opentelemetry.tracer.enabled:true}` will no longer show.